### DISCUSSION TITLE
hard to find codespaces in new home page layout

### DISCUSSION THREAD

**Discussion by ph-poppe** [community/community#58224] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hi there,

in the new home layout, it is hard to find the codespaces. It is no longer in any menu.

**Comment by Gustavonn07** [comment]
Hey to find codespace follow this steb by step:

<hr>

![image](https://github.com/community/community/assets/84361085/81930427-1cbd-4f0d-9399-77bf03f51751)

<hr>

![image](https://github.com/community/community/assets/84361085/388a515d-c0ff-46af-b808-f546cbec0376)

<hr>

![image](https://github.com/community/community/assets/84361085/1c3fd23b-6548-414a-abf3-85dae7a2b514)

<hr>

<br>

Fill in the information and that's it!

**Comment by ph-poppe** [comment]
it seems the new lay-out of the home page does no longer show that:

![image](https://github.com/community/community/assets/19587384/e3578b26-3b20-4e5d-861b-2891b63150a4)

**Comment by Gustavonn07** [comment]
My github updated now and I managed to find this one by clicking on my profile, maybe you already found it, but anything is here:

![image](https://github.com/community/community/assets/84361085/859d8bd8-08c6-461e-afab-64c18ccca6b4)

<hr>

![image](https://github.com/community/community/assets/84361085/4faf78ec-f5ef-40c1-9f2b-b11f1f7c2a37)
